bundle/edu.gemini.util.security/build.sbt

```diff
@@ -9,9 +9,18 @@ name := "edu.gemini.util.security"
 unmanagedJars in Compile ++= Seq(
   new File(baseDirectory.value, "../../lib/bundle/com-mchange-c3p0_2.10-0.9.5.jar"),
   new File(baseDirectory.value, "../../lib/bundle/scala-slick_2.10-1.0.0.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/scala-reflect-2.10.5.jar"),
   new File(baseDirectory.value, "../../lib/bundle/scala-swing_2.10-2.0.0-SNAPSHOT.jar"),
   new File(baseDirectory.value, "../../lib/bundle/scalaz-core_2.10-7.1.6.jar"),
-  new File(baseDirectory.value, "../../lib/bundle/scalaz-effect_2.10-7.1.6.jar"))
+  new File(baseDirectory.value, "../../lib/bundle/scalaz-concurrent_2.10-7.1.6.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/scalaz-effect_2.10-7.1.6.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/scalaz-stream_2.10-0.7.2a.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/doobie-core_2.10-0.2.4-SNAPSHOT.jar"),
+  new File(baseDirectory.value, "../../lib/bundle/shapeless_2.10-2.2.5.jar"),
```

Comment:
On the scala 2.11 branch we use the latest version of breeze which depends on shapeless 2.0, We'll have to check at integration time if there is conflict here